### ISSUE TITLE
The index on the right side of rails guides covers the main text, when the page is between 800 and 960 pixels wide.

### ISSUE THREAD

**Issue by arye-dov-eidelman** [rails/rails#33406]
### Steps to reproduce

1. Open any rails guide where the index (`#subCol`) flows past the general description on the top of the guide (`#feature`). i.e. just about any guide, for example [http://guides.rubyonrails.org/getting_started.html](http://guides.rubyonrails.org/getting_started.html)

2. Resize the window to between 800 and 960 pixels wide.

### Expected behavior

Two options:
1. The main text on the page (`#mainCol`) should reflow around the index (`#subCol`).

2. The media query stylus that applies up to 800 pixels should be extended to 960 pixels.

### Actual behavior

The main text on the page (`#mainCol`) is cut off on the right edge. (It's hidden behind the index (`#subCol`)).

### System configuration

**OS version**:
Windows 10.

**Browser version**:
Chrome, Firefox, Internet Explorer (7 - 11), Edge.

**Comment by twitnithegirl** [comment]
I'm seeing the same behavior but 960px seems like quite a large screen for the the sub 800px layout. I think we should either do a different media query for 960px and under or make that box more flexible the way the blue box is. As you are resizing the screen you see the text in the blue box changes breakpoints to accommodate the right menu. The main body text could do that.

**Comment by arye-dov-eidelman** [comment]
Somthing like this?
```css
@media screen and (min-width: 800px) and (max-width: 960px) {
  #container .wrapper {
    padding-right: 23em;
  }
}
```
I used 23 em because that's what the top (`#feature .wrapper`) has. But 21 seems large enough to me. 19.25 is the point where it overlaps.

**Comment by twitnithegirl** [comment]
That looks like it might be right. If you make a PR and attach it to this issue I'll pull it down and see how it looks.

**Comment by schneems** [comment]
Closed via PR #33645
